web/app_options.js

```diff
@@ -178,6 +178,14 @@ const defaultOptions = {
         : "../web/cmaps/",
     kind: OptionKind.API,
   },
+  standardFontDataUrl: {
```

Comment:
Please move this further down, since the parameters for *each* `OptionKind` should be sorted alphabetically.